miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleNumberPerProjectService.java

```diff
@@ -38,8 +38,8 @@ public void setSampleNumberPerProjectDao(SampleNumberPerProjectDao sampleNumberP
     this.sampleNumberPerProjectDao = sampleNumberPerProjectDao;
   }
 
-  public void setSqlProjectDAO(SQLProjectDAO sqlProjectDAO) {
-    this.sqlProjectDAO = sqlProjectDAO;
+  public void setSqlProjectDAO(HibernateProjectDao sqlProjectDAO) {
```

Comment:
Parameter and field type should be ProjectStore. Method and parameter name could be fixed too